### PR TITLE
Validation add missing descriptions

### PR DESCRIPTION
**Pull request by chrisms86**

### CODE REVIEW

File: iis/extensions/transform-manager/configurationmessage-constructor-string-string-string-string-microsoft-web-media-transformmanager.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: ConfigurationMessage Constructor (String, String, String, String) (Microsoft.Web.Media.TransformManager)
-description: Learn how the ConfigurationMessage constructor initializes a new instance of the ConfigurationMessage class using the specified name, file name, message, and help link ID. 
+description: This article contains information about syntax for the ConfigurationMessage constructor, with links to reference materials.
```
Comment (MeeraDi) [inline iis/extensions/transform-manager/configurationmessage-constructor-string-string-string-string-microsoft-web-media-transformmanager.md:3]: I would keep the original description only editing it to meet the character count requirement as: Learn how the ConfigurationMessage constructor initializes a new instance of the ConfigurationMessage class.

File: iis/extensions/url-rewrite/iis-url-rewrite.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: IIS URL Rewrite
+description: Describes IIS URL Rewrite, which implements URLs that are easier for users to remember and easier for search engines to find.
```
Comment (MeeraDi) [inline iis/extensions/url-rewrite/iis-url-rewrite.md:3]: Suggestion: Learn how the IIS URL Rewrite APIs allows you to create rules that define URL behavior.

File: iis/web-development-reference/native-code-api-reference/iprocesshost-interface.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "IProcessHost Interface"
+description: This article has information about syntax, methods, derived classes, inheritance hierarchy, and requirements for the IProcessHost interface.
```
Comment (MeeraDi) [inline iis/web-development-reference/native-code-api-reference/iprocesshost-interface.md:3]: For consistency: This article contains

**Comment by chrisms86** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**